Snakefile

```diff
@@ -9,11 +9,13 @@ SEMISUPERVISED, = glob_wildcards("model_configs/semi-supervised/{semisupervised}
 
 NUM_SEEDS = 3
 
-top_five_tissues = ['Blood', 'Breast', 'Stem_Cell', 'Cervix', 'Brain']
-
-combo_iterator = itertools.combinations(top_five_tissues, 2)
-TISSUE_STRING = ['.'.join(pair) for pair in combo_iterator]
+recount_top_five_tissues = ['Blood', 'Breast', 'Stem_Cell', 'Cervix', 'Brain']
+combo_iterator = itertools.combinations(recount_top_five_tissues, 2)
+RECOUNT_TISSUE_STRING = ['.'.join(pair) for pair in combo_iterator]
 
+gtex_top_five_tissues = ['Blood', 'Brain', 'Skin', 'Esophagus', 'Blood_Vessel']
```

Comment:
Are these the top five in terms of sample count?

Reply:
Yep! There isn't really a study equivalent in GTEx (that I know of) so I went with sample count